File: csharp/leap/Leap.cs
using System;

public static class Leap
{
    public static bool IsLeapYear(int year)
    {
        if (year % 100 == 0 && year % 400 != 0) {
            return false;
        }

        if (year % 4 != 0) {
            return false;
        }

        return true;
    }
}

File: go/leap/leap.go
// Package leap contains the solution for the Leap exercise on Exercism.
package leap

// IsLeapYear indicates whether the given year is a leap year.
func IsLeapYear(year int) bool {
	return year % 400 == 0 || year % 100 != 0 && year % 4 == 0
}
